presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestInsertIntoHiveTable.java

```diff
@@ -56,47 +53,37 @@ public Requirement getRequirements(Configuration configuration)
         }
     }
 
-    private static class AllSimpleTypesKnownToPrestoTables
-            implements RequirementsProvider
-    {
-        @Override
-        public Requirement getRequirements(Configuration configuration)
-        {
-            return compose(
-                    mutableTable(ALL_HIVE_SIMPLE_TYPES_KNOWN_TO_PRESTO_TEXTFILE, TABLE_NAME, CREATED),
-                    immutableTable(ALL_HIVE_SIMPLE_TYPES_KNOWN_TO_PRESTO_TEXTFILE));
-        }
-    }
-
-    @Test(groups = {HIVE_CONNECTOR, QUARANTINE})
+    @Test(groups = {HIVE_CONNECTOR})
     @Requires(AllSimpleTypesTables.class)
     public void testInsertIntoValuesToHiveTableAllHiveSimpleTypes()
     {
         String tableNameInDatabase = mutableTablesState().get(TABLE_NAME).getNameInDatabase();
         assertThat(query("SELECT * FROM " + tableNameInDatabase)).hasNoRows();
         query("INSERT INTO " + tableNameInDatabase + " VALUES(" +
-                "127, " +
-                "32767, " +
+                "TINYINT '127', " +
+                "SMALLINT '32767', " +
                 "2147483647, " +
                 "9223372036854775807, " +
-                "123.345, " +
                 "234.567, " +
+                "CAST(346 as DECIMAL(10,0))," +
+                "CAST(345.67800 as DECIMAL(10,5))," +
                 "timestamp '2015-05-10 12:15:35.123', " +
                 "date '2015-05-10', " +
                 "'ala ma kota', " +
-                "'ala ma kota', " +
+                "'ala ma kot', " +
```

Comment:
The test was broken and unmaintained for a while (that's why was quarantined). I didn't think it's worth to break it into a separate commit.